pkg/kwokctl/runtime/kind/kind.yaml.tpl

```diff
@@ -33,7 +33,7 @@ nodes:
 
   kubeadmConfigPatches:
 
-{{ if or .EtcdExtraArgs .EtcdExtraVolumes }}
+{{ if or .EtcdExtraArgs .EtcdExtraVolumes .Verbosity }}
```

Comment:
Move to https://github.com/kubernetes-sigs/kwok/blob/730d50e32b7968c4377b285f97c67d4819e8b657/pkg/kwokctl/runtime/kind/kind.go#L47